### PR TITLE
Fans and Temperatures Module

### PR DESCRIPTION
**Pull request by tuxudo**

### CODE REVIEW

File: app/views/listing/displays.php [hunk 1]
```diff
@@ -24,7 +24,7 @@
 			<th data-i18n="listing.displays.model" data-colname='displays.model'></th>
 			<th data-i18n="serial" data-colname='displays.display_serial'></th>
 			<th data-i18n="listing.power.manufactured" data-colname='displays.manufactured'></th>
-			<th data-i18n="listing.displays.nativeresolution" data-colname='displays.native'</th>
+			<th data-i18n="listing.displays.nativeresolution" data-colname='displays.native'></th>
```
Comment (bochoven) [inline app/views/listing/displays.php]: Please put this in a separate PR

**Comment by bochoven** [comment]
I wonder how future proof the 'one column per sensor' is.

**Comment by tuxudo** [comment]
It's fairly future restive. Apple reuses the more standard ones like battery, CPU, and hard drive ones. As new machines are released with new unique sensors, they can be added on as a new column. At the very least, a new machine that has new sensors will still have the basic temperatures listed
